### PR TITLE
Fix debugger issues 2.0

### PR DESCRIPTION
**Pull request by chipsenkbeil**
Same as #1527, but for the 2.0 branch.

### CODE REVIEW

**Comment by chipsenkbeil** [comment]
Not sure how I have @rorygraves commit on my PR. I just checked out my branch and rebased on top of 2.0.

**Comment by rorygraves** [comment]
Umm. I'm not sure either. That was on the 1.0 branch

**Comment by chipsenkbeil** [comment]
Is it needed for 2.0? Otherwise, I'll have to remove it somehow.

**Comment by fommil** [comment]
it's a small commit, I'm happy merging it with the larger one.

**Comment by fommil** [comment]
LGTM

File: project/EnsimeBuild.scala [hunk 1]
```diff
@@ -28,7 +28,7 @@ object EnsimeBuild extends Build {
        "io.spray" %% "spray-json" % "1.3.2"
     ),
 
-    // disabling shared memory gives a small performance boost to tests
+      // disabling shared memory gives a small performance boost to tests
```
Comment (fommil) [inline project/EnsimeBuild.scala]: you introduced some whitespace here, not a big deal

**Comment by fommil** [comment]
I am really looking forward to the emacs support for repl/debugging to improve because this is just going to **shine**

File: project/EnsimeBuild.scala (hunk 1 above)
Comment (chipsenkbeil) [inline project/EnsimeBuild.scala]: Do you want me to fix it? Or leave it? If I fix it, we're going to wait for all of the tests to run again.

**Comment by rorygraves** [comment]
Happy my commit sneaked in here - happy for it to be merged.

**Comment by chipsenkbeil** [comment]
Just waiting for Travis to actually run. >_>

**Comment by fommil** [comment]
take that travis

**Pull request merged**